### PR TITLE
chore: bump to latest ember

### PR DESCRIPTION
**Pull request by makepanic**
this bumps ember and fixes broken tests and linting issues that come from previously using jquery.

### CODE REVIEW

**Comment by makepanic** [comment]
I ran `npx ember-cli-update --to latest` which caused:

* all the docs to use npm so I removed the yarn lock and let it regenerate one for npm
* changed engines to node `"8.* || >= 10.*"`. #234 mentioned that it's preferred to drop 8. Should i remove it?

**Comment by sandstrom** [comment]
ping @rwjblue @stefanpenner I'm guessing you're the only ones with commit bit here? 

I was about to open a PR to fix https://github.com/emberjs/ember-inflector/issues/155, but I think it would make sense to get this PR in first (if possible + you think its a good idea)

File: package.json [hunk 1]
```diff
@@ -5,50 +5,58 @@
...
     "test:all": "ember try:each"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "8.* || >= 10.*"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.6.0"
+    "ember-cli-babel": "^7.13.0",
+    "ember-cli-htmlbars": "^4.2.0"
```
Comment (rwjblue) [inline package.json:31]: We don't have any templates (AFAIK), can you move this back into `devDependencies`?

**Comment by sandstrom** [comment]
friendly ping @makepanic 😄

**Comment by sandstrom** [comment]
friendly ping @makepanic 🌻

**Comment by rwjblue** [comment]
@sandstrom - If you have the time, would you mind taking over this PR?

**Comment by sandstrom** [comment]
@rwjblue Maybe, but I can't promise.

***

On a somewhat related note, when the new ember addon structure land, it would be good if there was less maintenance for addons. 

This addon have a bunch of files that doesn't add value, but still must be kept "up to date". For example `/app`, `/config`, `/vendor`, `/tests/dummy`, `.bowerrc`, etc. The fewer number of files required for a working addon, the less work it'll be to maintain it.

I think this is a good example of that.

cc @ef4

**Comment by stefanpenner** [comment]
I believe this has been superseded by https://github.com/emberjs/ember-inflector/pull/305/files

**Pull request closed without merging**